src/main/scala/ch/epfl/bluebrain/nexus/admin/marshallers/instances.scala

```diff
@@ -0,0 +1,126 @@
+package ch.epfl.bluebrain.nexus.admin.marshallers
+
+import akka.http.scaladsl.marshalling.GenericMarshallers.eitherMarshaller
+import akka.http.scaladsl.marshalling._
+import akka.http.scaladsl.model.MediaTypes.`application/json`
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.model.StatusCodes._
+import ch.epfl.bluebrain.nexus.admin.CommonRejection
+import ch.epfl.bluebrain.nexus.admin.CommonRejection.DownstreamServiceError
+import ch.epfl.bluebrain.nexus.admin.config.AppConfig.orderedKeys
+import ch.epfl.bluebrain.nexus.admin.config.Contexts._
+import ch.epfl.bluebrain.nexus.admin.resources.ResourceRejection
+import ch.epfl.bluebrain.nexus.admin.resources.ResourceRejection._
+import ch.epfl.bluebrain.nexus.commons.http.JsonLdCirceSupport.OrderedKeys
+import ch.epfl.bluebrain.nexus.commons.http.RdfMediaTypes._
+import ch.epfl.bluebrain.nexus.commons.http.syntax.circe._
+import ch.epfl.bluebrain.nexus.commons.types.HttpRejection
+import ch.epfl.bluebrain.nexus.rdf.Iri
+import ch.epfl.bluebrain.nexus.rdf.Iri.AbsoluteIri
+import ch.epfl.bluebrain.nexus.rdf.syntax.circe.context._
+import ch.epfl.bluebrain.nexus.service.http.directives.StatusFrom
+import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport
+import io.circe._
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.semiauto.deriveEncoder
+import io.circe.syntax._
+
+import scala.collection.immutable.Seq
+
+object instances extends FailFastCirceSupport {
+
+  private val rejectionConfig: Configuration = Configuration.default.withDiscriminator("code")
+
+  implicit val absoluteIriEncoder: Encoder[AbsoluteIri] = Encoder.encodeString.contramap(_.asString)
```

Comment:
You can import `import ch.epfl.bluebrain.nexus.rdf.instances._` and drop the `Encoder[AbsoluteIri]` and the `Decoder[AbsoluteIri]`

Reply:
for that you need rdf library 0.2.28